vm/src/obj/objnone.rs

```diff
@@ -39,6 +42,57 @@ impl PyNoneRef {
     fn bool(self, _vm: &mut VirtualMachine) -> PyResult<bool> {
         Ok(false)
     }
+
+    fn get_attribute(self, name: PyStringRef, vm: &mut VirtualMachine) -> PyResult {
```

Comment:
This piece I do not get. Do we want to remove it in future again?

Reply:
What do you mean again? We may be able to remove this in the future once we have slots implemented for the core operations. As we currently don't, we have no way of representing Option<PyObjectRef> as a PyValue, without conflating Some(PyNone) with None. So I had to find a workaround for cases where it mattered, which are properties on None.